docs/OPENAI_API.md

```
@@ -0,0 +1,1298 @@
+    event: "$ai_usage",
+    properties: {
+      model,
+      input_tokens: inputTokens,
+      output_tokens: outputTokens,
+      total_tokens: inputTokens + outputTokens,
+      cost,
+      request_id: requestId
+    }
+  });
+
+async function trackUsage(env: Env, { model, inputTokens, outputTokens, userId, requestId }: UsageEvent) {
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 106

(MD013, line-length)